CMakeLists.txt

```diff
@@ -278,6 +266,25 @@ include (CheckCXXCompilerFlag)
 # check the capabilities of the system we are building for
 #
 
+# Check for PAPI::SDE
+if( PARSEC_PROF_TRACE )
+  if(CMAKE_SYSTEM_NAME MATCHES "Windows" OR
+     CMAKE_SYSTEM_NAME MATCHES "MSYS" OR
+     CMAKE_SYSTEM_NAME MATCHES "MINGW64")
```

Comment:
I don't think you need to test for these systems, just leave the find_package to do its job (and be unable to link against SDE).